NomaiGrandPrix/SpawnPoints.tsv

```diff
@@ -45,109 +59,95 @@ Spawn_DreamZone_3_SecretGallery	DreamZone 3 Secret Gallery	DreamZone	TRUE	FALSE
-Spawn_JellyDomain	Giant's Deep Jelly Domain	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_GravityCannon	Gravity Cannon	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLE_Below	High Energy Lab Below	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLE_Interior	High Energy Lab Interior	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLE_Outside	High Energy Lab Outside	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLEPath	High Energy Lab Path	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_HT	Hourglass Twins (Orbit)	EmberTwin	FALSE	FALSE	FALSE	FALSE	
 Spawn_LakebedCave	Lakebed Cave	EmberTwin	FALSE	TRUE	TRUE	FALSE	
-SPAWN_MigrationPath	Migration Path		FALSE	TRUE	TRUE	FALSE	
-Spawn_Module_Broken	Orbital Probe Cannon Broken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
-Spawn_Module_Intact	Orbital Probe Cannon Intact Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
-Spawn_Module_Sunken	Orbital Probe Cannon Sunken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_QuantumMaze	Quantum Maze (Wandering Rock)	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_QuantumMoonLocator	Quantum Moon Locator	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_SandslideCave	Sandslide Cave	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_City	Sunless City	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_PodPath	Sunless City Trail	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_WarpReceiver	Warp Receiver	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_BrambleIsland	Bramble Island	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_ConstructionYard	Construction Yard	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Core	Core	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_GabbroIsland	Gabbro's Island	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_GD	Giant's Deep (Orbit)	GiantsDeep	FALSE	FALSE	FALSE	FALSE	
+Spawn_DeepSpace	Giant's Deep (Orbit)	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_JellyDomain	Jelly Domain	GiantsDeep	FALSE	FALSE	FALSE	FALSE	Duplicate
+Spawn_JellyDomain	Jelly Domain	GiantsDeep	FALSE	TRUE	TRUE	FALSE	Northern Half of GD
+Spawn_OPC	Probe Cannon	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Module_Broken	Probe Cannon Broken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Module_Intact	Probe Cannon Intact Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Module_Sunken	Probe Cannon Sunken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_StatueIsland_Beach	Statue Island Beach	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_StatueIsland_Inside	Statue Island Inside	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_StatueIsland_Top	Statue Island Top	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_QuantumIsland_Bottom	Tower of Quantum Trials	GiantsDeep	FALSE	TRUE	TRUE	FALSE	Outside by elevator
+Spawn_QuantumIsland_PuzzleRoom4	Tower of Quantum Trials Bottom	GiantsDeep	FALSE	TRUE	TRUE	FALSE	End of last solved puzzle
+Spawn_QuantumIsland_Top	Tower of Quantum Trials Top	GiantsDeep	FALSE	TRUE	TRUE	FALSE	Before entering first puzzle
+Spawn_CO	Interloper	Interloper	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_CO	Interloper (Orbit)	Interloper	FALSE	FALSE	FALSE	FALSE	
```

Comment:
Should we call this (and similar spawns) "Interloper Orbit" instead without the parens? If we have "Interloper Core" and "Interloper Interior," "Interloper Orbit" follows the same pattern

Reply:
Everything with "Orbit" is deactivated for now anyway (Unless I missed one), they're all ship spawns which seemed to be duplicates or out in space which i thought was uninteresting